clang-tools-extra/clang-tidy/readability/FunctionSizeCheck.cpp

```diff
@@ -39,6 +41,9 @@ class FunctionASTVisitor : public RecursiveASTVisitor<FunctionASTVisitor> {
     if (!Node)
       return Base::TraverseStmt(Node);
 
+    if (IgnoreMacros && Node->getBeginLoc().isMacroID())
+      return true;
```

Comment:
could we check the whole Stmt is macro here. It will cause some unexpected behavior when the stmt is start with macro.
btw. I am not sure whether we should let macro increase statement count.